app/src/main/res/values-de/strings.xml

```diff
@@ -330,6 +330,7 @@
     <string name="help">Hilfe</string>
     <string name="pref_next_up_behavior_title">Informationen zum nächsten Video anzeigen</string>
     <string name="lbl_time_range">%1$s–%2$s</string>
+    <string name="lbl_random">Zufällig</string>
```

Comment:
Translations should be added via translate.jellyfin.org and never via GitHub. Please remove this change from the git history of this pull request.